coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -3210,6 +3213,10 @@ func auditActionFromTransition(transition database.WorkspaceTransition) database
 }
 
 type TemplateVersionImportJob struct {
+	// TemplateID is not guaranteed to be set. Template versions can be created
+	// without being associated with a template. Resulting in a template id of
+	// `uuid.Nil`
+	TemplateID         uuid.UUID                `json:"template_id"`
```

Comment:
Should this be a `uuid.NullUUID` then?

Reply:
I guess it should be. The `uuid.NullUUID` is just annoying to use, but is more accurate.